prepare/templates/qa/multiple_choice/templates.py

```diff
@@ -0,0 +1,104 @@
+from src.unitxt.templates import TemplatesDict, TemplatesList
+from src.unitxt.blocks import InputOutputTemplate
+from src.unitxt.catalog import add_to_catalog
+
+templates = {
+    "mmlu": """The following are multiple choice questions (with answers) about {topic}.\n{sentence1}.\nAnswers: {choices}.\nAnswer:""".strip(),
+    "helm": """The following are multiple choice questions (with answers) about {topic}.\n\nQuestion: {sentence1}.\nAnswers: {choices}.\nAnswer:""".strip(),
+    "lm_eval_harness": """Question: {sentence1}.\nChoices:\n{choices}.\nAnswer:""".strip(),
+    "fm_eval": """The following are multiple choice questions (with answers) about {topic}.\n\nQuestion: {sentence1}\nChoose from {numbers}\nAnswers: {choices}\nAnswer:""".strip(),
+}
+
+
+# MMLU_TEMPLATES = TemplatesDict(
+#     {key: InputOutputTemplate(input_format=val, output_format="{label}", postprocessors=["processors.first_character"]) for key, val in templates.items()}
+# )
+
+for k, v in templates.items():
+    template = InputOutputTemplate(input_format=v, output_format="{label}", postprocessors=["processors.first_character"])
+    add_to_catalog(template, f"templates.qa.multiple_choice.original.{k}", overwrite=True)
+
+# with context
+
+
+def replace_if_context_not_there(s, oldvalue, newvalue):
+    if "{context}" in s:
+        return s
+    else:
+        return s.replace(oldvalue, newvalue)
+
+
+templates_with_context = {
```

Comment:
i must say this is a convulated way to define the templates (inherited in from past code).
Wouldn't it be simpler and more readible, just to write the 4 templates as they are expected to appear?